.github/runners/self-hosted/update-runners.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+pushd $(dirname $0) > /dev/null
+
+runner_user="manager"
+
+if [ ! -e runners.conf ]; then
+    echo "ERROR: Could not find runners.conf"
+    exit 1
+fi
+
+for runner_ip in $(cat runners.conf); do
+    echo "runner_ip=${runner_ip}"
+
+    echo "syncing actions-runner-hooks:"
+    rsync -av --progress --stats --delete actions-runner-hooks ${runner_user}@${runner_ip}:
+
+    echo "listing actions-runner-hooks:"
+    ssh ${runner_user}@${runner_ip} "ls -l ~/actions-runner-hooks"
+
+    echo "synching .env:"
+    rsync -av --progress --stats .env ${runner_user}@${runner_ip}:~/actions-runner/.env
+
+    echo "displaying .env:"
+    ssh ${runner_user}@${runner_ip} "cat ~/actions-runner/.env"
+
+    echo "restarting the runner:"
+    ssh ${runner_user}@${runner_ip} "sudo reboot"
```

Comment:
I did not know we reboot

Reply:
This is only for when new hooks and/or hook configurations are installed on the self hosted runners.  Up until now I've been doing this on my own with a completely private set of scripts (which these are based on) so nobody besides me knew what was happening (or when).  I would very carefully only do this when no jobs were running, so everyone was completely unaware unless you happened to be looking at the runner status (https://github.com/organizations/perftool-incubator/settings/actions/runners) and happened to briefly see them go into offline status instead of idle.